go/config/config.go

```diff
@@ -344,6 +345,7 @@ func newConfiguration() *Configuration {
 		GraphiteConvertHostnameDotsToUnderscores:     true,
 		GraphitePollSeconds:                          60,
 		URLPrefix:                                    "",
+		MaxOutdatedKeysToShow:                        10,
```

Comment:
Please make the default larger (say `64`), and set your own personal config to `10`